ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -120,6 +120,10 @@ def IdentityN(*, input, name="IdentityN"):
     return [ivy.copy_array(x) for x in input]
 
 
+def Inv(*, x, name="Inv"):
+    return ivy.astype(ivy.divide(ivy.ones_like(x), x), x.dtype)
```

Comment:
we already have `ivy.reciprocal` function which can be simply used instead of a composition